Parking-Lot/src/com/practice/parkinglot/CommonUtils.java

```diff
@@ -0,0 +1,15 @@
+package com.practice.parkinglot;
+
+class Pair<F, S> {

```

Comment:
Why naming it Pair? Keep it very specific to what it is storing. Also, I don't think a separate class was required here.

Reply:
This entire file is over-engineering and should be avoided in the machine coding round in the interest of time.